free-1.md

```diff
@@ -6,63 +6,92 @@ good way to meet their needs.
 
 
 ### What's the need?
-_What need is met by your idea? Who are you helping? What is that person's
-experience like now? What would their experience be like if you could help 
-them?_
+Currently, it is difficult for a non-technical user to create a personal
+website with enough flexibility. Also, it is difficult for such a user to 
```

Comment:
Well, it's difficult for them to do it on their own; there are services for this, but I agree it could be good to enable people to do it themselves if they so chose.